gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/ApiBuilder.java

```diff
@@ -88,6 +88,7 @@ public ApiBuilder services(Services services) {
     public Api build() {
         var api = new Api();
         api.setDefinitionVersion(definitionVersion);
+        api.setExecutionMode(ExecutionMode.V3);
```

Comment:
why not adding `paths` and `flows` ? (as you did for flows on v4 side of `ApiBuilder`)

Reply:
Because as it was v2 definition, I was lazy 🙈 
Currently, this builder acts as a builder and a fixture. We should split those responsibilities by using Lombok for the builder and creating a test-jar with fixtures. I won't do it here and now, but I will keep in mind 🙈